<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaPrincipal</class>
 <widget class="QMainWindow" name="JanelaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>238</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>397</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>446</width>
    <height>291</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Relógio Binário</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLCDNumber" name="lcdNumber_4">
          <property name="digitCount">
           <number>2</number>
          </property>
          <property name="intValue" stdset="0">
           <number>16</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_3">
          <property name="digitCount">
           <number>2</number>
          </property>
          <property name="intValue" stdset="0">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="digitCount">
           <number>2</number>
          </property>
          <property name="intValue" stdset="0">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_2">
          <property name="digitCount">
           <number>2</number>
          </property>
          <property name="intValue" stdset="0">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_5">
          <property name="digitCount">
           <number>2</number>
          </property>
          <property name="intValue" stdset="0">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="edt_h_16">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edt_h_8">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edt_h_4">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edt_h_2">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edt_h_1">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>11</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>        Mostrador</string>
          </property>
          <widget class="QRadioButton" name="rb_data">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>106</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Data</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="rb_hora">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>106</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Hora</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnsobre">
          <property name="text">
           <string>&amp;Sobre</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLCDNumber" name="lcdNumber_7">
          <property name="digitCount">
           <number>2</number>
          </property>
          <property name="intValue" stdset="0">
           <number>32</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edt_m_32">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLCDNumber" name="lcdNumber_8">
          <property name="digitCount">
           <number>2</number>
          </property>
          <property name="intValue" stdset="0">
           <number>16</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edt_m_16">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLCDNumber" name="lcdNumber_12">
          <property name="digitCount">
           <number>2</number>
          </property>
          <property name="intValue" stdset="0">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edt_m_8">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLCDNumber" name="lcdNumber_11">
          <property name="digitCount">
           <number>2</number>
          </property>
          <property name="intValue" stdset="0">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edt_m_4">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLCDNumber" name="lcdNumber_10">
          <property name="digitCount">
           <number>2</number>
          </property>
          <property name="intValue" stdset="0">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edt_m_2">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLCDNumber" name="lcdNumber_9">
          <property name="digitCount">
           <number>2</number>
          </property>
          <property name="intValue" stdset="0">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edt_m_1">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
